src/core/mcir/common.go

```diff
@@ -2032,6 +2047,23 @@ func UpdateResourceObject(nsId string, resourceType string, resourceObject inter
 
 }
 
+func expandInfraType(infraType string) string {
+	expInfraTypeList := []string{}
+
```

Comment:
```suggestion
infraType := strings.ToLower(infraType)

```
혹시 모르니, 소문자 기준 비교를 하면 좋을 것 같습니다. _